brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-874 Improvement: Letter spacing in all block.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-874

### Screenshots

- Screenshot - https://share.getcloudapp.com/d5uO0n8n
- Screencast - https://share.getcloudapp.com/mXuxq2ZB

### Types of changes




- New feature (non-breaking change which adds functionality)

### How has this been tested?

- Drag and drop any block. For ex. Icon List.
- Go to style > Label > click on typography you will see the Letter spacing option.
- Add letter spacing and check it is rendering in the editor as well as on the front.
- It should work fine on both.

### Checklist:
